### DISCUSSION TITLE
Why Aren't Any Updates This Month?

### DISCUSSION THREAD

**Discussion by sr229** [vignetteapp/vignette#117] [Announcements]
Hey all! Thanks for your support and believing in the Vignette Project. As you may have noticed, there hasn't been any kind of activity, and you may wonder why? This community post will tackle those one by one on why we haven't made updates this January.

## Why We Haven't Posted Anything In a Month?

I'm pretty sure you're aware of my ranting in [Twitter recently](https://twitter.com/satominyaaa/status/1352491186209005570), and yes, I do get hate mail for not delivering a usable build of Vignette. Let me ascertain that we are still committed to build this project for everyone, but due to circumstances, we could not do it. Here's a detailed explanation why:

- The project lead (me: @sr229), and @LeNitrous have zero infrastructure to work on. My laptop is broken beyond repair after the storm that hit the Philippines last year had it soaked with all my work gone. A kind soul told me there may be a hardware I can lease by February but there's no concrete timeframe, and it's not assured it'll come as well, giving uncertainty to our project.

- We simply lack the funds to do so. We can build this project but this kind of project is very financially straining, I (Ayane) had to finance this myself using my own wallet, and it has reached to the point I had to suspend financing it to save up for other expenses. The technology is something we can do but it takes considerable money to make this technology optimized and reachable enough to you folks.

- We're also waiting for key technology changes in osu!framework. Right now making the UI is engineering-wise very expensive to do. There's no fancy layouting like XAML in osu!framework which makes it hard to engineer the product. The backend is something we can do but the UI is just too much effort to do.

- And we simply have zero time now. Nathan and I are currently College students. We do this on our free time, which brings us on why the Twitter post was made. We simply don't have the time and resources to make this project work. We want to get a stable job firsthand but we won't forget about this project as we believe in it's potential.

## Closing Words

I would like to thank you personally @smoogipoo for the first and only sponsor of the project, he believed on this project and with that in mind, I promise we won't give up on making this a working product. I myself want to make this work but we're simply deprived of things we need to make this work.

Rest assured, I will do my best to release this, we just need more time, with that in mind, we won't give out any release dates to make sure we deliver the best kind of product possible. As always, thank you for your undying support.

See you again on another update! I hope this is the last time I would make this kind of call for help, but I figured it was necessary since we kept this issue under wraps for a while. Let us know in the comments if you have any more questions.

**Comment by raynei86** [comment]
sadge
:RushiaCry:

**Reply by katsuko9** [reply]
you can do it!
while I cannot support you financially yet, I will still follow the project.
I believe in the finition of Vignette (´；ω；｀)
がんばれ！乗り越え！

**Comment by Squiddu** [comment]
Seems like good progress is being made now, though!
It still seems like it'll be a while, UX elements and face tracking are big things!
